CHANGELOG.md

```diff
@@ -4,6 +4,33 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 1.11.0
+### Added 
+* Service: Added support for sharding corpus storage accounts using "Premium" storage accounts for improved IOPs.  [#334](https://github.com/microsoft/oneufzz/pull/334)
+* CLI/Service/Agent: Added the ability to optionally colocate multiple similar tasks on a single machine. The coverage and crash reporting tasks in the LibFuzzer template make use of this functionality by default. [#402](https://github.com/microsoft/oneufzz/pull/402)
+* CLI: Added `onefuzz debug log tail` which enables continuously following Application Insights query results.  [#401](https://github.com/microsoft/oneufzz/pull/401)
+* CLI/Agent: Add support to optionally verify LibFuzzer targets execute correctly at the start of a task using `-help=1`, which will enable identifying non-functional LibFuzzer targets.  [#381](https://github.com/microsoft/oneufzz/pull/381)
+* CLI/Agent: Add support to optionally specify if LibFuzzer targets exit with non-zero error codes without crashing inputs as a warning rather than a failure.  [#381](https://github.com/microsoft/oneufzz/pull/381)
```

Comment:
This sentence is a little hard to parse, I think we want something like:
```suggestion
* CLI/Agent: Support specifying whether to log a warning or fail the task when a LibFuzzer target exits with a non-zero status code (without also generating a crashing input).  [#381](https://github.com/microsoft/oneufzz/pull/381)
```

Reply:
Added